flink-runtime/src/test/java/org/apache/flink/runtime/rest/RestEndpointITCase.java

```diff
@@ -162,8 +225,8 @@ public void testEndpoints() throws Exception {
 
 		@Override
 		protected CompletableFuture<TestResponse> handleRequest(@Nonnull HandlerRequest<TestRequest, TestParameters> request, RestfulGateway gateway) throws RestHandlerException {
-			Assert.assertEquals(request.getPathParameter(JobIDPathParameter.class), PATH_JOB_ID);
-			Assert.assertEquals(request.getQueryParameter(JobIDQueryParameter.class).get(0), QUERY_JOB_ID);
+			assertEquals(request.getPathParameter(JobIDPathParameter.class), PATH_JOB_ID);
```

Comment:
How about we stick to the import pattern that already exists in a given file instead of switching back and forth?

Reply:
Sure.